public/css/downloads.css

```
@@ -110,7 +110,14 @@ nav a {
   display: block;
   margin: 30px auto 0 auto;
   width: 100%;
+  border-radius: 15px;
```

```suggestion
  border-radius: 5px;
```
I personally like how the dropdown looks currently but I'm willing to compromise if the border-radius is brought down to make it less round.